src/python/pants/backend/scala/lint/scalafmt/BUILD

```
@@ -0,0 +1,8 @@
+# Copyright 2021 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+python_sources(dependencies=[":resources"])
+
+python_tests(name="tests")
+
+resources(name="resources", sources=["*.lockfile.txt"])
```

Same feedback as Google Java Format:

```suggestion
resource(name="lockfile", source="scalafmt.default.lockfile.txt")
```